packages/discord.js/src/structures/Webhook.js

```diff
@@ -207,7 +207,10 @@ class Webhook {
 
     const { body, files } = await messagePayload.resolveFiles();
     const d = await this.client.rest.post(Routes.webhook(this.id, this.token), { body, files, query, auth: false });
-    return this.client.channels?.cache.get(d.channel_id)?.messages._add(d, false) ?? d;
+    return (
+      this.client.channels?.cache.get(d.channel_id)?.messages._add(d, false) ??
+      new (require('./Message').Message)(this.client, d)
```

Comment:
Do `let Message;` above and then here (and everywhere else) `(Message ??= require('./Message').Message)`.